src/components/contextual/pages/pool/add-liquidity/composables/useMyWallet.ts

```diff
@@ -0,0 +1,89 @@
+import { isSameAddress } from '@/lib/utils';
+import { useTokens } from '@/providers/tokens.provider';
+
+import useInvestPageTabs, {
+  Tab,
+  tabs,
+} from '@/composables/pools/useInvestPageTabs';
+import { AmountIn, useJoinPool } from '@/providers/local/join-pool.provider';
+import { usePool } from '@/providers/local/pool.provider';
+
+export function useMyWallet() {
+  /**
+   * COMPOSABLES
+   */
+  const { pool, isLoadingPool } = usePool();
+  const { setAmountsIn, setJoinWithNativeAsset, isSingleAssetJoin, amountsIn } =
+    useJoinPool();
+  const { nativeAsset, wrappedNativeAsset, getMaxBalanceFor } = useTokens();
+  const { activeTab } = useInvestPageTabs();
+
+  /**
+   * COMPUTED
+   */
+  const excludedTokens = computed<string[]>(() => {
```

Comment:
don't understand this, why we put pool address to `excluded tokens`, maybe problem in naming here?